tests/compile/index.html

```diff
@@ -22,7 +22,7 @@
 <body>
   <h1>Blockly: Advanced Compilation Test</h1>
 
-  <p>To run this test manually, run `npm run test:compile:advanced`
+  <p>To run this test manually, run `node tsick.js` and `npm run test:compile:advanced`
```

Comment:
Can you make the `test:compile:advanced` command also run `tsick.js`?

Reply:
I didn't integrate it here because I think it would make more sense to have the `build:js` command run `tsick.js`.  There is nothing disruptive in its changes.  Doing so would allow `build:compressed` to switch to advanced optimizations once we have an externs list.

That's why I'm proposing loosely storing tsick here for now, without integrating it into the flow.

Reply:
I'd rather integrate it directly now, so that running the test does the full workflow, and then possibly pulling it up into the `build:js` command down the line. It's relatively straightforward, and makes it harder to run tests in invalid ways.

Reply:
The difficulty is that I will soon introduce a change that causes `test:compile:advanced` to run `build:js` automatically, preventing `tsick.js` from being run manually in between.

However, I think it better to commit this PR in (approximately) its current form and then I will integrate it into the pipeline as needed.